hwdb.d/60-sensor.hwdb

```diff
@@ -824,7 +824,16 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:rvnPOV:rnI102A:*
 sensor:modalias:i2c:bmc150_accel:dmi:bvnINSYDECorp.:*:svnInsyde:pnBayTrail:*:rvn105B:rn0E57:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
+##########################################
+# Positivo
 #########################################
+
+#Positivo Duo K116R
```

Comment:
Please insert a space between `#` and comment.

Reply:
Added  a space between # and comment.